.github/workflows/main.yml

```diff
@@ -22,102 +22,100 @@ on:
 
 env:
   OCTOVERSION_CurrentBranch: ${{ github.head_ref || github.ref }}
-  OCTOVERSION_Patch: ${{ github.run_number }}
```

Comment:
Without this override the version is solely determined by OctoVersion, which computes the same thing every night unless new git tags are added. We timestamp the nightlies so that's not a problem